src/modules/SambaAPI.py

```diff
@@ -2,6 +2,22 @@
 import_module('PackageSystem')
 import_module('Package')
 from yast import PackageSystem, Package
+if not PackageSystem.Installed('samba-python3'):
+    if Package.InstallAll(['samba-python3']):
+        from samba.net import Net
+        from samba.credentials import Credentials
+        from samba.dcerpc import nbt
+    else:
+        ycpbuiltins.y2error('Failed to import samba python bindings')
```

Comment:
maybe something like
``` diff
    else:
-        ycpbuiltins.y2error('Failed to import samba python bindings')
+        raise ImportError("Failed to install samba python bindings samba-python3")
```

That should work, should dump appropriate trace into y2log etc.
note: you will need https://github.com/yast/yast-python-bindings/pull/26
Then you won't need to check variiou